graph/typeDefs.graphql

```diff
@@ -52,14 +52,30 @@ type User {
   status: USER_STATUS
 }
 
```

Comment:
We can create something like this:

```
enum TaggableModels {
  COMMENT,
  USER,
  ASSET
}
```

To control which models can receive comments.